### ISSUE TITLE
[Accessibility] No H1 heading is defined in the 'Graph Explorer' page.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1903]
**Test Environment:**
OS Build: Version 22H2(OS Build 25136.1000) 
Browser: Version 104.0.1278.2 (Official build) dev (64-bit)
URL: [Graph Explorer | Try Microsoft Graph APIs - Microsoft Graph](https://developer.microsoft.com/en-us/graph/graph-explorer)
Tool- Accessibility insight for web
Screen reader: Narrator 

**Repro Steps:**
1. Open above [URL ](https://developer.microsoft.com/en-us/graph/graph-explorer)[i](https://developer.microsoft.com/en-us/microsoft-teams/app-development-resources)n Edge dev browser.
2. Turn on Narrator then scan mode using Caps+ Space key.
3. Navigate to headings on the page by pressing the 'h' in scan mode and also check with Accessibility insight tool.
4. Observe the issue here.

**Actual Result:**
No H1 heading is defined in the 'Graph Explorer' page.

**Expected Result:**
'Graph Explorer' text on the page should be defined under H1 heading tag programmatically.

**User Impact:** 
Screen reader users will be confused with the structure of page if no H1 heading is defined programmatically on the page.
 
**MAS Reference:**
[Info and Relationships - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01)

**Additional context**
Issue: https://office.visualstudio.com/OC/_workitems/edit/6195406
